### ISSUE TITLE
Links in messages should inherit color (if any)

### ISSUE THREAD

**Issue by xPaw** [thelounge/lounge#72]
for example, if you send `{color color green}http://example.com`, the link will remain blue, not green as dictated by the color code.

I'm thinking we should have:

```
#chat a {
    color: inherit;
    text-decoration: inherit;
}
```

This will make links match color of the text in all cases. Which is not a bad idea, as links won't be as eye catchy.

**Comment by xPaw** [comment]
This also happens with underline decoration which is overwritten.

**Comment by astorije** [comment]
:+1: to respect colors set in an IRC message
:-1: to have links to match text color when no colors are specifically set in IRC messages

**Comment by PolarizedIons** [comment]
Is this still an issue? This works for me ![](https://i.imgur.com/o0A8ZAh.png)

**Comment by astorije** [comment]
Alright, I had quite the brain fart trying to play with colors on #thelounge-spam, only to realize colors don't work there.
But testing in another channel, indeed this seems fixed, and it looks like it was fixed in 2.4.0 as well!

**Comment by YaManicKill** [comment]
Hurrah for accidental bug fixes!
